app/components/cards/collectionCard.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+import React from "react";
+import {
+  Card,
+  CardHeading,
+  Text,
+  CardContent,
+  Tooltip,
+} from "@nypl/design-system-react-components";
+import styles from "../swimlanes/Swimlanes.module.css";
```

Comment:
Ah, start creating separate CSS module files for each component. So there should be a new one for `CollectionCard` to better scope styles to each component.